docs/sources/developers/plugins/development-with-local-grafana.md

```diff
@@ -74,25 +78,27 @@ Finally start your plugin in development mode. Go to your plugin root directory
    yarn start
    ```
 
-   2.2 Or with docker-compose, in your plugin directory, run:
+   b. Or, with docker-compose in your plugin directory, run:
 
    ```bash
    docker-compose up
    ```
 
-After this, you should be able to see your plugin listed in Grafana and test your changes. Note that any change in the fronted will require you to refresh your browser while changes in the backend may require to rebuild your plugin binaries and reload the plugin (`mage && mage reloadPlugin` for local development or `docker-compose up` again if you are using docker-compose).
+After this, you should be able to see your plugin listed in Grafana, and then you can test your changes. 
+
+If you make a change in the fronted, you must refresh your browser. However, changes in the backend may require that you rebuild your plugin binaries and reload the plugin (`mage && mage reloadPlugin` for local development, or run `docker-compose up` again if you are using docker-compose).
 
 ## Run your backend plugin with a debugger
 
 > Note: The following method only works with a local Grafana instance and currently doesn't work with Docker.
 
-You can run a backend plugin and attach a debugger to it, which allows you to set breakpoints and debug your backend plugin directly from your IDE of choice.
+Running a backend plugin with a debugger is a feature that supports Visual Studio Code and GoLand out of the box, but it can also work with any other IDE or debugger.
 
-We support Visual Studio Code and GoLand out of the box, but this feature can also work with any other IDE or debugger.
+You can run a backend plugin and attach a debugger to it, which allows you to set breakpoints and debug your backend plugin directly from your IDE of choice:
 
 1. Go to your plugin's folder.
 
-1. Check your `go.mod` and make sure `grafana-plugin-sdk-go` is at least on `v0.156.0`
+1. Make sure that your `go.mod` and `grafana-plugin-sdk-go` are at least on `v0.156.0`
```

Comment:
This rewrite doesn't quite make sense, you are checking a file `go.mod` for a line like https://github.com/grafana/grafana-plugin-examples/blob/main/examples/app-with-backend/go.mod#L5 to check the SDK is at a specific version

Reply:
Thanks. I have changed to "Check your `go.mod` to make sure...". I misunderstood because "and" is a bit ambiguous.